torch/onnx/_constants.py

```diff
@@ -2,7 +2,7 @@
 
 ONNX_ARCHIVE_MODEL_PROTO_NAME = "__MODEL_PROTO"
 onnx_default_opset = 14
-onnx_main_opset = 17
+onnx_main_opset = 18
```

Comment:
We can consider adding an "experimental_opset"

Reply:
The root cause for the failure is that ORT does not support opset 18. a new `onnx_next_opset = 18` wouldn't help PyTorch itself or any other engine, but ORT only. So a new constant here would be dead weight.

Better keep using 17 until I push a PR to ONNX Runtime setting 18 as the max opset. That could happen if we change https://github.com/microsoft/onnxruntime/pull/12311 from contrib to ONNX.

Reply:
Set as experimental, we can enable behaviors like warning users that it is a version that is not released?